packages/xdg_directories/test/xdg_directories_test.dart

```diff
@@ -113,6 +113,28 @@ XDG_VIDEOS_DIR="$HOME/Videos"
     xdg.xdgProcessManager = const LocalProcessManager();
   });
 
+  test('Returns null when xdg-user-dir executable is not present', () {
+    xdg.xdgProcessManager = FakeProcessManager(
+      <String, String>{},
+      canRunExecutable: false,
+    );
+    final List<String> userDirVariables = <String>[
```

Comment:
This test doesn't need to run for a bunch of different inputs since the behavior isn't input-dependent; you can just call it one with one arbitrary string.

Reply:
Fixed!